File: lua/test/exists_test.lua
-- MixpanelLexiconSchemas SDK exists test

local sdk = require("mixpanel-lexicon-schemas_sdk")

describe("MixpanelLexiconSchemasSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// MixpanelLexiconSchemas SDK exists test

require_once __DIR__ . '/../mixpanellexiconschemas_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = MixpanelLexiconSchemasSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}
